### PR TITLE
Addded xbox one controller to the list

### PR DESCRIPTION
**Pull request by wlwatkins**
Hope this helps, there is only one main modification which is theconfig json where two ids where switched between the 360 and the one

### CODE REVIEW

File: platforms/joystick/joystick_driver.go [hunk 1]
```diff
@@ -25,6 +25,10 @@ const (
 
 	// Xbox360RockBandDrums controller configuration.
 	Xbox360RockBandDrums = "xbox360RockBandDrums"
+	
+	// Xbox360 joystick configuration.
+	xboxONE = "xboxONE"
```
Comment (krobertson) [inline platforms/joystick/joystick_driver.go:30]: This needs to be changed to `XboxONE` as the constant name.

**Comment by deadprogram** [comment]
Also, needs `go fmt` run on the changed files, please.

**Comment by gen2thomas** [comment]
done now by #871 , thanks for your work and sorry for the long delay

**Comment by wlwatkins** [comment]
Wow this i had forgotten about ^^

**Pull request closed without merging**